// api/mockPhones.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "Apple iPhone 5c",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 823,
    image: "https://drop.ndtv.com/TECH/product_database/images/910201410645AM_635_apple_iphone_6_plus.jpeg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "2",
    categoryId: "1",
    name: "Apple iPhone 6",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 953,
    image: "https://store.storeimages.cdn-apple.com/4667/as-images.apple.com/is/image/AppleInc/aos/published/images/H/C8/HC804/HC804?wid=572&hei=572&fmt=jpeg&qlt=95&op_usm=0.5,0.5&.v=1478212606357",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "3",
    categoryId: "4",
    name: "Lenovo A6000",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 764,
    image: "https://drop.ndtv.com/TECH/product_database/images/1102015124741PM_635_lenovo_a6000.jpeg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "4",
    categoryId: "5",
    name: "Nokia Lumia 1520",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 674,
    image: "https://drop.ndtv.com/TECH/product_database/images/1022201314344PM_635_nokia-Lumia-1520.jpeg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "5",
    categoryId: "3",
    name: "HTC One",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 674,
    image: "https://cdn.techjuice.pk/wp-content/uploads/2017/05/HTC-One-M8.jpg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "6",
    categoryId: "2",
    name: "Samsung Galaxy S6",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 674,
    image: "https://drop.ndtv.com/TECH/product_database/images/312015113532PM_635_samsung_galaxy_s6_gold_platinum.jpeg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  }
]
